sylviawanjiku/Store_manager_Api
# #161360489 Test user endpoints

#### What does this PR do?
Create tests for user registration and login (post methods)
#### Description of Task to be completed?
Create a model for the user data
Create an add-user method for registering the user
Create a method to get all users and from there a method to check if user exists and allow user to login and if not the user cannot be allowed to login 
#### How should this be manually tested?
In postman the create user endpoint should allow you to post a new user and then the get method for the endpoint should allow you to get the list of the users
The login endpoint should check if user exists if so the user should login successfully if not user should not be able to login and gets a message wrong credentials cannot login
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#161360489
https://www.pivotaltracker.com/story/show/161360489
#### Screenshots (if appropriate)
#### Questions:
